tools/AzureMarkdownRewriterTool/AzureVideoRawInformation.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Microsoft.DocAsCode.Tools.AzureMarkdownRewriterTool
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Newtonsoft.Json;
+
+    public class AzureVideoRawInformation
+    {
+        [JsonProperty("namePropertyName")]
+        public string NamePropertyName { get; set; }
+
+        [JsonProperty("displayNamePropertyName")]
+        public string DisplayNamePropertyName { get; set; }
+
+        [JsonProperty("collectionNamePropertyName")]
+        public string CollectionNamePropertyName { get; set; }
+
+        [JsonProperty("pollingInterval")]
+        public int PollingInterval { get; set; }
+
+        [JsonProperty("fastPollingInterval")]
+        public int FastPollingInterval { get; set; }
+
+        [JsonProperty("slowPollingInterval")]
+        public int SlowPollingInterval { get; set; }
+
+        [JsonProperty("isComplete")]
+        public bool IsComplete { get; set; }
+
+        [JsonProperty("partialErrors")]
+        public string PartialErrors { get; set; }
+
+        [JsonProperty("data")]
+        public List<AzureVideoDataItem> Data { get; set; }
+    }
+
+    public class AzureVideoDataItem
+    {
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [JsonProperty("title")]
+        public string Title { get; set; }
+
+        [JsonProperty("submissionStatus")]
+        public string SubmissionStatus { get; set; }
+
+        [JsonProperty("published")]
+        public DateTime Published { get; set; }
```

Comment:
Keep same with Azure's json data.

---
In reply to: [73270153](https://github.com/dotnet/docfx/pull/559#discussion_r73270153) [](ancestors = 73270153)